lib/interpret_date/date_type.rb

```diff
@@ -0,0 +1,11 @@
+require "interpret_date"
```

Comment:
I don't think this require is necessary but I do think we need to require this file in `lib/interpret_date.rb`.